docs/ibexa_cloud/ddev_and_ibexa_cloud.md

```diff
@@ -6,59 +6,63 @@ description: Use DDEV to run an Ibexa Cloud project locally.
 
 Two ways are available to run an [[= product_name_cloud =]] project locally with DDEV:
 
-- [by using the Platform.sh's `ddev-platformsh` add-on](#with-the-ddev-platformsh-add-on)
-- [like other existing project, without this add-on](#without-the-platformsh-add-on).
+- [by using the Platform.sh's `ddev-platformsh` and `ddev-ibexa-cloud` add-ons](#with-the-ddev-platformsh-and-ddev-ibexa-cloud-add-ons)
+- [like other existing project, without this add-on](#without-the-ibexa-cloud-add-on).
 
 !!! note
 
     The following examples use [[[= product_name_cloud =]] CLI (`ibexa_cloud`)](https://cli.ibexa.co/).
 
-## With the `ddev-platformsh` add-on
+## With the `ddev-platformsh` and `ddev-ibexa-cloud` add-ons
 
-To configure the [`ddev/ddev-platformsh` add-on](https://github.com/ddev/ddev-platformsh), you need a [Platform.sh API Token](https://docs.platform.sh/administration/cli/api-tokens.html).
+To configure [`ddev/ddev-platformsh` add-on](https://github.com/ddev/ddev-platformsh) and [`ddev/ddev-ibexa-cloud` add-on](https://github.com/ddev/ddev-ibexa-cloud), you need a [Platform.sh API Token](https://docs.platform.sh/administration/cli/api-tokens.html).
 
 The `ddev/ddev-platformsh` add-on configures the document root, the PHP version, the database, and the cache pool according to the [[= product_name_cloud =]] configuration.
 About the search engine, the add-on can configure Elasticsearch but can't configure Solr.
 If you use Solr on [[= product_name_cloud =]] and want to add it to your DDEV stack, see [Clustering with DDEV and `ibexa/ddev-solr` add-on](clustering_with_ddev.md#solr).
 
-`COMPOSER_AUTH` from Platform.sh can't be used, because JSON commas are incorrectly interpreted by `--web-environment-add`, which sees them as multiple variable separators.
+The `ddev/ddev-ibexa-cloud` add-on integrate the `ibexa_cloud` command,
+and eases the pull of cloud contents into the local installation.
+
+`env:COMPOSER_AUTH` from Platform.sh can't be used, because JSON commas are incorrectly interpreted by `--web-environment-add`, which sees them as multiple variable separators.
 But the variable must exist for Platform.sh `hooks` scripts to work.
 To use an `auth.json` file for this purpose, see [Using `auth.json`](install_with_ddev.md#using-authjson).
 
 You must remove Node.js and NVM installations as they're already included in DDEV.
```

Comment:
I know you're not changing this, but this part was not clear to me (I'm not sure what it means and which action I should do)

Reply:
Doesn't seem needed anymore since https://github.com/ibexa/post-install/commit/26edf2733ffe07565565bc9657dabdde9f130121

I'll simply remove this note. I didn't encounter any Node.js/NVM conflict with DDEV's ones.

Reply:
Removed in 32b6fe7b23a9d3fef8ac89f677a441c8787a736b